extensions-contrib/parquet-extensions/src/main/java/io/druid/data/input/parquet/GenericRecordAsMap.java

```diff
@@ -74,7 +74,7 @@ public boolean containsValue(Object value)
    * <ul>
    * <li>null, boolean, int, long, float, double, string, Records, Enums, Maps, Fixed -> String, using String.valueOf</li>
    * <li>bytes -> Arrays.toString() or new String if binaryAsString is true</li>
-   * <li>Arrays -> List&lt;String&gt;, using Lists.transform(&lt;List&gt;dimValue, TO_STRING_INCLUDING_NULL)</li>
```

Comment:
It shouldn't be changed, as well as all other comments, changed in this commit.

Reply:
*sigh* intellij... fixed